operations/research-and-development/product/pm-onboarding.md

```diff
@@ -0,0 +1,240 @@
+  * [Customer Feedback](https://community.mattermost.com/private-core/channels/customer-feedback)
+  * [Customer Support](https://community.mattermost.com/private-core/channels/community)
+* Begin watching training videos on [academy.mattermost.com ](https://academy.mattermost.com/)
+* Begin reading [docs.mattermost.com](https://docs.mattermost.com/)
+
+Team:
+* Meet the rest of team leads \(Developers, Designer, QA\) and join team meetings
+  * Ask questions to understand development, design, QA processes
+* Meet with Amy \(Release Manager\) and read the release process doc
+* Meet with Ian (CEO) \[[https://github.com/it33/readme/blob/master/README.md](https://github.com/it33/readme/blob/master/README.md/)\). This will be scheduled for you
+* Meet other Product Managers \(Jason Blais, Eric Sethna, Katie Wiersgalla, Aaron Rothschild, Ian Tao, Chen-I Lim\)
+  * Learn about their areas of ownership
+* Participate in PM, R&D, and Customer Obsession Meetings
+* Jira
+  * Sign up to Jira and post in the “Jira configuration” channel to be added to internal team
+  * Get Jira training from PM buddy
+    * [Mattermost Platform](https://mattermost.atlassian.net/projects/MM/issues/MM-10977?filter=allopenissues)
+* Make your first edit and pull request on GitHub
+  * Update documentation or product help text
+* Small contributions
+  * File a bug in Jira
+  * Test a new feature on a test server and suggest an improvement
+  * Leave feedback on a design proposal
+
+Readings:
+* The Advantage:
+  * [https://drive.google.com/file/d/1zOV4nqVPxRHsQrxcqQwkGq2hCvB2CKJU/view](https://drive.google.com/file/d/1zOV4nqVPxRHsQrxcqQwkGq2hCvB2CKJU/view)
+* The Five Dysfunctions of a Team: A Leadership Fable
+  * [https://drive.google.com/file/d/1yvLMW5tHGoTTjrq5VY17SCoeQoNXn9jA/view?usp=sharing](https://drive.google.com/file/d/1yvLMW5tHGoTTjrq5VY17SCoeQoNXn9jA/view?usp=sharing)
+
+## First 30 days
+
+### Objective: Build foundational knowledge
+
+### Training focus:
+
+* Users and customers
+* Product
+  * Features/Functions
+  * Technical
+* Market and Competitive Landscape
+* Team
+  * People and Processes
+
+### Activities:
+Learn about our customers and users
+* Support
+  * Customers: Zendesk \(Sven, Support Lead)\)
+    * Meet with Sven for Zendesk training and support overview
+    * Join support standups for a week or two, and help out on support tickets
+  * Community: GitHub, Forums \(Amy\)
+* Customer Success \(Meet with Jenn\)
+  * Overview of Customer Success team, who owns what
+  * Current Customer Success processes
+  * Typical challenges during customer onboarding
+  * Join one CSM team meeting to meet the rest of the CS team
+* Customer calls
+  * Join calls with CSMs
+  * Join calls with Sales team
+* Feature requests
+  * [Uservoice forums](https://mattermost.uservoice.com/)
+  * [Process feedback in productboard](https://mattermost.productboard.com/insights/shared-inbox) 
+* Learn about the market
```

Comment:
Any specific steps or links to learn about the market?

Reply:
Updating the formatting to make the steps here more obvious.